README.md

```diff
@@ -175,7 +175,6 @@ find es-vue-base/src/ | entr -s 'npm --prefix es-vue-base run build'
 
```

Comment:
Similarly not a huge fan of markdown formatting, but I think I'm in the minority here